lib/vector-core/src/metrics/handle.rs

```diff
@@ -253,6 +253,7 @@ mod test {
     }
 
     #[test]
+    #[allow(clippy::needless_pass_by_value)]
```

Comment:
It seems like we could resolve this one via making `inner` below take `value: &[f64]`?

Reply:
We could, except that quickcheck can't make an arbitrary `&[f64]`.

Reply:
Documented in 2c56044